py/mock.py

```diff
@@ -390,7 +390,8 @@ def setup_logging(config_path, config_opts, options):
 
     try:
         # set up logging format strings
-        config_opts['build_log_fmt_str'] = log_cfg.get("formatter_%s" % config_opts['build_log_fmt_name'], "format", raw=1)
+        config_opts['build_log_fmt_str'] = log_cfg.get("formatter_%s"\
+        % config_opts['build_log_fmt_name'], "format", raw=1)
```

Comment:
The 2nd line should be indented, and I don't think the backslash is needed. At least when splitting on the comma before "format" which would look like a better place for the split.